airbyte-server/src/main/java/io/airbyte/server/converters/OperationsConverter.java

```diff
@@ -95,4 +96,62 @@ public static OperationRead operationReadFromPersistedOperation(final StandardSy
         .operatorConfiguration(operatorConfiguration);
   }
 
+  private static OperatorWebhook webhookOperatorFromConfig(io.airbyte.api.model.generated.OperatorWebhook webhookConfig) {
+    final var operatorWebhook = new OperatorWebhook().withWebhookConfigId(webhookConfig.getWebhookConfigId());
+    if (webhookConfig.getWebhookType() == null) {
```

Comment:
for understanding: when is this null?

Reply:
When the client is sending the old format (i.e., the deprecated fields).

Reply:
client = FE?

Reply:
this is something we can remove once the FE is migrated over?

Reply:
Yes indeed!

Reply:
Nice! Do we have a clean up ticket so we don't forget?